dracut/90stratis-clevis/module-setup.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/bash
+# This dracut module requires the kernel command line parameter rd.neednet=1 if
```

Comment:
Has this comment become obsolete or are we sticking with the neednet=1 requirement?

Reply:
This is still necessary but not in all cases so it is up to the user to determine if they need it (only when using Tang).